src/feature/marketplace/ui/marketplace.uielement.ts

```diff
@@ -11,12 +11,16 @@ import {
   Image,
   isBusy,
   PageHeaderTile,
+  path,
   Row,
   Span,
   UiElement,
 } from '@earnkeeper/ekp-sdk';
+import { userCard } from 'src/util/user-card';
```

Comment:
Is this being used? Importing from `src/util/user-card` will cause an error. I think this line can be removed?

Reply:
yeah its not useful.